services/web/server/src/simcore_service_webserver/login/_constants.py

```diff
@@ -64,6 +64,9 @@
     str
 ] = "Invalid code. Please provide valid code or generate new code."
 MSG_WRONG_2FA_CODE__EXPIRED: Final[str] = "Expired code. Please generate new code."
+MSG_WRONG_CAPTCHA__INVALID: Final[
+    str
+] = "The CAPTCHA code entered was incorrect. Please try again."
```

Comment:
- Does the user knows about what the CAPTCHA is? Is it mentioned in the front-end?
- Should this error message include information on how to re-generate the code by clicking the image?

NOTE that alternatively the texts for the error messages could also be composed in the front-end. I wonder whether that would make more sense in a near future so the front-end could be faster adapted to the user's language and also released faster. Then the backend errors would also provide with the minimal information to both detect what happens and elements to compose the message

Reply:
discussed